GliaWidgets/Glia.swift

```diff
@@ -122,9 +122,66 @@ public class Glia {
     public var onEvent: ((GliaEvent) -> Void)?
 
     private var rootCoordinator: RootCoordinator?
+    private var interactor: Interactor?
 
     private init() {}
 
+    /// Setup SDK using specific engagement configuration without starting the engagement.
+    /// - Parameters:
+    ///   - configuration: Engagement configuration.
+    ///   - queueID: Queue identifier.
+    ///   - visitorContext: Visitor context.
+    public func configure(
+        with configuration: Configuration,
+        queueID: String,
```

Comment:
I'd like to propose to use camelcase properly and rename `queueID` property to `queueId`.

Reply:
Renamed 8b8964479107402cfc1109e51405c49891bc43d5.